uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -252,9 +370,17 @@ sub determineSubjectID {
 }
 
 
-################################################################
-################### createTarchiveArray ########################
-################################################################
+=pod
+
+=head3 createTarchiveArray($tarchive, $globArchiveLocation)
+
+Creates the tarchive information hashref.
+
+INPUT: tarchive's path, globArchiveLocation option
```

Comment:
you mean `globrchiveLocation` is optional?
I believe for optional inputs, you had a standard way of mentioning them in one of the lower methods in this file. Maybe you can pick a convention and use it?

Reply:
not really, this refers to the option globArchiveLocation specified as an argument of several scripts